interfaces/seccomp/backend.go

```diff
@@ -162,6 +162,75 @@ func bpfBinPath(srcName string) string {
 	return filepath.Join(dirs.SnapSeccompDir, strings.TrimSuffix(srcName, ".src")+".bin")
 }
 
+func parallelCompile(compiler Compiler, profiles []string) error {
+	if len(profiles) == 0 {
+		// no profiles, nothing to do
+		return nil
+	}
+
+	profilesQueue := make(chan string, len(profiles))
+	N := runtime.NumCPU()
+	if N >= 2 {
+		N -= 1
+	}
+	if N > len(profiles) {
+		N = len(profiles)
+	}
+	res := make(chan error, N*2)
```

Comment:
nitpick, looking at my suggestion in full,  if  N = len(profiles), then here we could just use N instead of N*2.  One way to do that is to have this size as M and set it appropriately along N.